src/main/groovy/com/blackducksoftware/integration/hub/detect/DetectConfiguration.groovy

```diff
@@ -182,7 +182,7 @@ class DetectConfiguration {
         }
         configurationPieces.add('-'.multiply(60))
         configurationPieces.add('')
-        String configurationMessage = configurationPieces.join(System.lineSeparator())
+        String configurationMessage = configurationPieces.join('\n')
```

Comment:
Why dont we use the System.lineSeparator here? Switching to \n seems inappropriate here